src/index.html

```diff
@@ -5,9 +5,52 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="product"
+      data-qa="card"
+    >
+      <div class="product__image"></div>
+      <h2 class="product__title">
+        APPLE A1419 iMac 27" Retina
+        <br>
+        5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="product__code">
+        Product code: 195434
+      </p>
+      <div class="product__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="product__rating_reviews">
+          Reviews: 5
+        </p>
+      </div>
+      <div class="product__price">
```

Comment:
Add a blank line between the blocks everywhere
```suggestion
      </div>
      
      <div class="product__price">
```